DEVELOPING.md

```diff
@@ -6,14 +6,16 @@ This document explains how to build, test, and develop features for revive.
 
 Clone the project:
 
-```
+```sh
 git clone git@github.com:mgechev/revive.git
 cd revive
 ```
+
 ## Build
 
 In order to build the project run:
-```bash
+
+```sh
```

Comment:
Why updating bash to sh?

I'm fine with it, as they are aliases in linguist (the lib used by GitHub to highlight code)

But I'm curious if markdown-lint made you change this